modules/models/src/main/java/org/eclipse/xpanse/modules/models/billing/ServicePrice.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.billing;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Data;
+import org.eclipse.xpanse.modules.models.billing.enums.Currency;
+import org.eclipse.xpanse.modules.models.billing.enums.PricingPeriod;
+
+/**
+ * Defines the price of the managed service.
+ */
+@Data
+public class ServicePrice {
+
+    @NotNull
+    @Schema(description = "The final amount of the price.")
+    private BigDecimal amount;
+
+    @NotNull
+    @Schema(description = "The discount amount of the price.")
+    private BigDecimal discountAmount;
+
+    @NotNull
+    @Schema(description = "The official amount of the price.")
+    private BigDecimal officialAmount;
```

Comment:
this might be something specific to Huaweicloud. For now, it should be enough if we stick to just one price for now - the actual price the customer has to pay after the deducting all discounts.

Reply:
Ok, I'll just keep the field `amount` to show the final actual price.